firefly_client/firefly_client.py

```diff
@@ -1453,6 +1454,64 @@ def rvstring_from_dict(self, rvdict):
                               gamma_value = rvdict['gamma_value'])
         return rvstring
 
+    # -----------------------------------------------------------------
+    # image line based footprint overlay
+    # -----------------------------------------------------------------
+    def overlay_imagepixel_based_footprint_layer(self, footprint_data, title=None,
```

Comment:
I would like to shorten the method name. My proposal is `overlay_footprints`. The coordinate system of the footprints is part of the `footprint_data`